File: intro/isBeautifulString.py
# https://app.codesignal.com/arcade/intro/level-10/PHSQhLEw3K2CmhhXE
def isBeautifulString(s):
    a = "abcdefghijklmnopqrstuvwxyz"
    for i in range(1, len(a)):
        if s.count(a[i-1]) >= s.count(a[i]):
            continue
        return False
    return True


print(isBeautifulString("bbbaacdafe"))

File: intro/isBeautifulString.js
// https://app.codesignal.com/arcade/intro/level-10/PHSQhLEw3K2CmhhXE
function isBeautifulString(inputString) {
    s = "abcdefghijklmnopqrstuvwxyz"
    for (i = 1; i < s.length; i++) {
        if (inputString.split(s[i]).length - 1 > inputString.split(s[i - 1]).length - 1) {
            return false
        }
    }
    return true
}
console.log(isBeautifulString('bbc'))